### DISCUSSION TITLE
Elapsed time

### DISCUSSION THREAD

**Discussion by CsatiZoltan** [neperfepx/FEPX#18] [Q&A] [answered]
Hi,

FEPX prints the start time of the simulation in the header, such as `Start time: 2021-5-27 at 16:12`
It would be very useful if the elapsed time would also be shown
- in stdout and/or
- in a separate file

Thanks,
Zoltan

**Comment by mkasemer** [comment, marked as answer]
Hi @CsatiZoltan : Thank you for the suggestion! This feature is already implemented in the development branch and will be pushed to public in the next release (coming soon). Elapsed time will print to console in units seconds at the end of the simulation. Example:

```
Info   :       Loads on Z0:  -0.145716E+01 -0.198013E+00  0.124535E+00
Info   :       Loads on Z1:  -0.345621E+00 -0.316437E+01  0.835105E-06
Info   : Elapsed time:   752.698 secs.
Info   : Final step terminated. Simulation completed successfully.
========================================================================
```

**Reply by CsatiZoltan** [reply]
Perfect! Where can I access the development branch? I can only see `master` and `gh-pages`.

**Reply by mkasemer** [reply]
Sorry, I meant the development repo, which is private. Please keep an eye out for the next release, which should be coming shortly.
